include/openenclave/internal/crypto/cert.h

```diff
@@ -345,6 +345,20 @@ oe_result_t oe_get_crl_distribution_points(
     uint8_t* buffer,
     size_t* buffer_size);
 
+/**
+ * Gets the validation datetimes from the certificate.
+ *
+ * @param cert[in] the certificate.
```

Comment:
[in] should be on the @param not on the "cert" part, i.e.:
@param[in] cert The certificate.

Reply:
Opened #2205 for this.